dockers/docker-fpm-frr/frr/bgpd/templates/sentinels/policies.conf.j2

```diff
@@ -0,0 +1,18 @@
+!
+! template: bgpd/templates/sentinels/policies.conf.j2
+!
+{% if constants.bgp.sentinel_community is defined %}
```

Comment:
is it possible sentinel_community not defined?

Reply:
sentinel_community is defined in constants.yaml file, this item should exist as it is also firstly added in this commit, add this for a safer check.